UPDATING.md

```diff
@@ -10,4 +10,17 @@ EHRbase 2.0.0 comes with a completely overhauled data structure that is not auto
 new version over an older data structure.
 
 To support the migrating of data from systems `pre-2.0.0` to `2.0.0`, a migration tool and instructions are provided 
-at https://github.com/ehrbase/migration-tool. 
\ No newline at end of file
+at https://github.com/ehrbase/migration-tool. 
+
+
+## EHRbase 2.7.0
+
+### EHR_STATUS and FOLDER consistency check
+
+Updating an `EHR_STATUS` or `FOLDER` did not check the `If-Match header` against the DB. This allowed to pass an uid 
+contained header that does not match the existing uid in the DB.
+This may have lead to inconsistent data in some systems. A manual migration script is provided to, first check if a
```

Comment:
We will not provide the migration script

Reply:
oops ... removed